notifier.rst

```diff
@@ -66,6 +66,7 @@ OvhCloud    ``symfony/ovhcloud-notifier``     ``ovhcloud://APPLICATION_KEY:APPLI
 Sinch       ``symfony/sinch-notifier``        ``sinch://ACCOUNT_ID:AUTH_TOKEN@default?from=FROM``
 Smsapi      ``symfony/smsapi-notifier``       ``smsapi://TOKEN@default?from=FROM``
 Twilio      ``symfony/twilio-notifier``       ``twilio://SID:TOKEN@default?from=FROM``
+Sendinblue  ``symfony/sendinblue-notifier``   ``sendinblue://API_KEY@default?sender=PHONE``
```

Comment:
Please sort them alphabetically. Thanks

Reply:
Done @OskarStark thanks for your review :)